### PR TITLE
Adds finger snapping

### PR DESCRIPTION
**Pull request by Hardly3D**
## About The Pull Request
This PR adds the finger snapping emote, ported from Beestation (and NSV), https://github.com/BeeStation/BeeStation-Hornet/pull/1208 and https://github.com/BeeStation/NSV13/pull/338

https://user-images.githubusercontent.com/66234359/203344246-6f42a23f-93bd-4dfc-9c1c-e1324dadd5d4.mp4
## Why It's Good For The Game
Many uses for it, such as snapping at assistants and getting someone's attention.
Also RP
## Changelog

:cl: Cenrus, ported by Hardly
add: Added *snap emote
/:cl:

### CODE REVIEW

**Comment by Imaginos16** [comment]
INSANELY BASED

**Comment by xPokee** [comment]
Based

**Comment by Oranggu** [comment]
I honestly cant think of a reason this should not be added lol

**Comment by JohnFulpWillard** [comment]
> I honestly cant think of a reason this should not be added lol

lack of gbp sorry

**Comment by Oranggu** [comment]
> lack of gbp sorry

Shits fucked

**Comment by Wallemations** [comment]
The sound effect should have you snap twice in a row so you can do advanced Intruder RP

**Comment by datorangebottle** [comment]
Snapping repeatedly should get the attention of nearby pets. Except cats, who should sometimes run away from you if you snap repeatedly because cats are fickle like that.

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Move it to carbon and check if you have hands.

**Comment by Hardly3D** [comment]
Should be good for review now.

File: code/modules/mob/living/carbon/emote.dm [hunk 1]
```diff
@@ -134,6 +134,19 @@
...
+	key_third_person = "snaps"
+	message = "snaps their fingers."
+	message_param = "snaps their fingers at %t."
+	emote_type = EMOTE_AUDIBLE
+	hands_use_check = TRUE
+	muzzle_ignore = TRUE
+
+/datum/emote/living/carbon/snap/get_sound(mob/living/user)
+	if(ishuman(user))
+		return pick('sound/misc/fingersnap1.ogg', 'sound/misc/fingersnap2.ogg')
```
Comment (Mothblocks) [inline code/modules/mob/living/carbon/emote.dm:148]:
```suggestion
		return pick('sound/misc/fingersnap1.ogg', 'sound/misc/fingersnap2.ogg')
	return null
```

**Pull request merged**